kapitan/targets.py

```diff
@@ -408,7 +408,7 @@ def search_targets(inventory_path, targets, labels):
     return targets_found
 
 
-def compile_target(target_obj, search_paths, compile_path, ref_controller, inventory_path, **kwargs):
```

Comment:
`inventory_path` was never used inside the function and should be passed forward inside `**kwargs`